### DISCUSSION TITLE
email auth package server not found error

### DISCUSSION THREAD

**Discussion by abdullah517** [saran-surya/email_auth_node#59] [Q&A]
Heroku is not free for deployment now so I am using netlify but after deployment I can face issue 404 page not found

**Comment by saran-surya** [comment]
Actually kindly let me know if you can host backend using netlify.. ?

**Reply by abdullah517** [reply]
Actually I am beginner in this I am taking help from some videos

**Reply by abdullah517** [reply]
I am currently working on app and I want to verify user email using email auth package. This package gives error remote server not found. I followed instructions on how to resolve this issue from your git repository. But in deployment phase I am facing issue

**Reply by abdullah517** [reply]
I have watched some videos how to deploy website using netlify but I faced error 404 page not found

**Reply by saran-surya** [reply]
Thing is netlify will only support you for the frontend support, for backend as per the docs of netlify, you can use AWS lambda I believe for backend support.

Here is a clarification: https://stackoverflow.com/questions/66755427/can-i-run-a-front-end-and-back-end-on-netlify

**Comment by abdullah517** [comment]
Ok let me try
